src/m365/spo/commands/web/web-roleinheritance-break.ts

```diff
@@ -0,0 +1,91 @@
+import { Cli, Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+}
+
+class SpoWebRoleInheritanceBreakCommand extends SpoCommand {
+  public get name(): string {
+    return commands.WEB_ROLEINHERITANCE_BREAK;
+  }
+
+  public get description(): string {
+    return 'Break role inheritance of subsite';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+  }
+
```

Comment:
We are missing telemetry for this command.

Example:
https://github.com/pnp/cli-microsoft365/blob/f956c5da7fec2c629abd18947a88f6bd149f9974/src/m365/spo/commands/web/web-roleassignment-remove.ts#L42-L51

This method is called in the constructor.